automatic/kitty/tools/helpers.ps1

```diff
@@ -0,0 +1,20 @@
+﻿function StopProcesses() {
+  $processNames = @("genpass", "gensfc", "kageant", "cygtermd", "kitty", "kittygen", "klink", "kscp", "ksftp")
+  $processes = Get-Process  $processNames -ea SilentlyContinue
+
+  if ($processes) {
+    $activeProcesses = ($processes | Select-Object -ExpandProperty name | Sort-Object -Unique) -join ","
+    Write-Warning "Stopping processes: $activeProcesses to prevent upgrade/uninstall failure..."
+    Stop-Process -InputObject $processes
+
+    Start-Sleep -Seconds 3
```

Comment:
Fixed sleep is automation is generally antipattern and should almost never be used.

Reply:
Sleep removed